install/kubernetes/cilium/charts/agent/templates/clusterrole.yaml

```diff
@@ -56,6 +56,7 @@ rules:
   - customresourcedefinitions
   verbs:
   - create
+  - delete
```

Comment:
Let's drop this commit. If we don't need these permissions we shouldn't blindly add them.